### ISSUE TITLE
backend actions fail if configuration is not global

### ISSUE THREAD

**Issue by ebaschiera** [Affirm/Magento_Affirm#27]
Hi,
I noticed that in Payment Model, the method _apiRequest() doesn't take into account the website-store configuration layers.
If I set the Api key and Secret key at website level (let's say I configure them for one website only), leaving the global values blank, all actions perfomed in magento backend fail.

**Comment by affirmPE** [comment]
------------------------------------------------------------------------------------------------------------
Type your response on top of this email to respond to this case. Learn Email Commands Here - https://support.desk.com/customer/portal/articles/1191712-reply-via-email-commands
------------------------------------------------------------------------------------------------------------


Case Assignment Notification - Case #13991 ( https://affirm.desk.com/web/agent/case/13991 )

Affirm | Apr 26, 2017 02:52PM UTC

------------------------------------------------------------------------------------------------------------
Case #13991
Customer: Affirm/Magento_Affirm
Subject: [Affirm/Magento_Affirm] backend actions fail if configuration is not global (#27)
------------------------------------------------------------------------------------------------------------
Case Information (13991):

Subject: [Affirm/Magento_Affirm] backend actions fail if configuration is not global (#27)

Description: 
Status: Open
Priority: 4

Assigned Group: MTS



------------------------------------------------------------------------------------------------------------
Case History
------------------------------------------------------------------------------------------------------------
 goatmad  at APR 26, 2017 | 07:52AM PDT:
Hi,
I noticed that in Payment Model, the method _apiRequest() doesn't take into account the website-store configuration layers.
If I set the Api key and Secret key at website level (let's say I configure them for one website only), leaving the global values blank, all actions perfomed in magento backend fail.

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/Affirm/Magento_Affirm/issues/27

------------------------------------------------------------------------------------------------------------
This message was sent to reply@reply.github.com in reference to Case #13991
------------------------------------------------------------------------------------------------------------
